src/lib/LoginFile.js

```diff
@@ -102,28 +99,24 @@ class LoginFile {
 
     _drawWarningText() {
         const x = LoginFile.WIDTH / 2;
-        const y = LoginFile.HEIGHT - 32;
+        const y = LoginFile.HEIGHT - 86 - 2;
```

Comment:
If not doing `LoginFile.HEIGHT - 88` I would prefer a comment explaining the different parts of the calculation.

Reply:
Only because I don't want to write that number.